packages/taquito-core/src/error/common.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ *  @category Error
+ *  @description Error that indicates an invalid address being passed or used (both contract and implicit)
```

Comment:
Thinking with InvalidContractAddressError being distinguished that InvalidAddressError will just be implicit?